snippets/lwc-xml.json

```diff
@@ -0,0 +1,72 @@
+{
+    "Api Version": {
+        "body": [
+            "<apiVersion>$0</apiVersion>"
+        ],
+        "description": "Api Version",
+        "prefix": [
+            "apiVersion"
+        ],
+        "scope": "xml"
+    },
+    "Capability": {
+        "body": [
+            "<capability>${1|lightningCommunity__RelaxedCSP,lightning__ServiceCloudVoiceToolkitApi|}</capability>"
+        ],
+        "description": "Capability",
+        "prefix": [
+            "capability"
+        ],
+        "scope": "xml"
+    },
+    "Is Exposed": {
+        "body": [
+            "<isExposed>${1|true,false|}</isExposed>"
+        ],
+        "description": "Is Exposed",
+        "prefix": [
+            "isExposed"
+        ],
+        "scope": "xml"
+    },
+    "Targets": {
+        "body": [
+            "<targets>",
+            "\t$0",
```

Comment:
Here you can also add `targetConfig` tags, because `targets` never exist without `targetConfig`